src/constants/Error.js

```diff
@@ -0,0 +1,10 @@
+export const EMPTY_CAR_NAME_ERROR = '[ERROR] 빈 문자열을 입력할 수 없습니다.';
+export const ONE_CAR_NAME_ERROR =
+  '[ERROR] 경주할 자동차 이름을 두 개 이상 입력해주세요.';
+export const MORE_THAN_FIVE_LETTERS =
+  '[ERROR] 자동차 이름은 5글자를 넘어갈 수 없습니다.';
+export const SAME_CAR_NAME_ERROR = '[ERROR] 자동차 이름은 중복될 수 없습니다.';
+export const NOT_NUMBER = '[ERROR] 숫자가 아닌 값은 입력할 수 없습니다.';
+export const INVALID_NUMBER = '[ERROR] 시도횟수는 0 보다 큰 숫자여야 합니다.';
+export const TOO_BIG_NUMBER =
+  '[ERROR] 너무 큰 숫자를 입력했습니다. 20 이하의 숫자로 입력해야 합니다.';
```

Comment:
```suggestion
const ERROR = {
  EMPTY_CAR_NAME_ERROR : '<내용>',
  ONE_CAR_NAME_ERROR : '<내용>',
  
   --- 생략--- 
}
export default ERROR;
```

와 같이 하나의 객체에 담아 내보내는 것은 어떨까요??